viral/learning_stages.py

```
@@ -1,133 +1,877 @@
+                ]
+            )
+            for trial in session.trials
+            if trial_is_imaged(trial)
+            and (rewarded is None or trial.texture_rewarded == rewarded)
+        ]
+    )
+
+
+def tuning_comparison_plot() -> None:
+
+    1 / 0
```

Ruff (0.14.14)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)